templates/ingress.yaml

```diff
@@ -1 +1,37 @@
-# TODO: Research & add Ingress controller for exposing st2web service to public net (#6)
+{{- if .Values.ingress.enabled }}
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  {{- $name := printf "%s-%s" $.Release.Name .Values.ingress.name | trunc 63 | trimSuffix "-" | quote }}
```

Comment:
We don't have `Values.ingress.name` defined anywhere

Reply:
Having no `Values.ingress.name`, we can switch back to simplified templating similar to
https://github.com/StackStorm/stackstorm-ha/blob/2f92c2679948ca9bd7468e015571bfb1b96b5149/templates/services.yaml#L5